### DISCUSSION TITLE
API support for: "Multi-ControlNet / Joint Conditioning (Experimental)

### DISCUSSION THREAD

**Discussion by joseph-schwartz** [Mikubill/sd-webui-controlnet#313] [Ideas]
Is there API support for: "Multi-ControlNet / Joint Conditioning (Experimental)?" Been looking for how to do it. If not, is this the location to ask for support for this, or is there another place to request that? (This is amazing by the way, combining even two control nets has brought a level of control to this I can barely believe, awesome job!) - Posted this same thing in the API discussion board.

**Comment by kkiqq22** [comment]
"Combining Multiple ControlNets",This function is amazing. It is strongly recommended to update it to API mode
